pom.xml

```diff
@@ -386,6 +391,11 @@
       <version>2.0.2</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
What's the reason of having the version and scope declared in separate places?

Reply:
Fixed